go.mod

```diff
@@ -42,6 +42,7 @@ require (
 	github.com/tendermint/go-amino v0.15.1
 	github.com/tendermint/tendermint v0.34.0-rc3
 	github.com/tendermint/tm-db v0.6.2
+	github.com/zeebo/blake3 v0.0.4
```

Comment:
Blake3 indeed does seem like an improvement, but I'm always a bit wary about new and shiny tech that's "better, faster, stronger".

Some questions I have:
- Has any project (blockchain or not) used blake3 in production, and has some data/insights about it?
- Is this library audited? If not, are we okay to use this library from "zeebo"?

Reply:
I'm sure @ValarDragon would have excellent input here.

Reply:
Blake3 looks _amazing_ at first glance. And I tend to agree with @amaurymartiny. Would love to have those questions answered and hear what others think. Thanks for working on this @robert-zaremba !